cmd/lit-af/netcmds.go

```diff
@@ -164,3 +180,50 @@ func (lc *litAfClient) Say(textArgs []string) error {
 	fmt.Fprintf(color.Output, "%s\n", reply.Status)
 	return nil
 }
+
+func (lc *litAfClient) RemoteControlAuth(textArgs []string) error {
+	stopEx, err := CheckHelpCommand(rcAuthCommand, textArgs, 2)
+	if err != nil || stopEx {
+		return err
+	}
+	args := new(litrpc.RCAuthArgs)
+	reply := new(litrpc.StatusReply)
+
+	pubKey, err := hex.DecodeString(textArgs[0])
```

Comment:
hey, just trying this out, why do we use a hex encoded argument here? remnants of the old brontide stuff?